autonomy/cli/deploy.py

```diff
@@ -158,12 +163,25 @@ def build_deployment_command(  # pylint: disable=too-many-arguments, too-many-lo
 
     keys_file = Path(keys_file or DEFAULT_KEYS_FILE).absolute()
     build_dir = Path(output_dir or DEFAULT_ABCI_BUILD_DIR).absolute()
+
     packages_dir = Path(packages_dir or Path.cwd() / "packages").absolute()
     open_aea_dir = Path(open_aea_dir or Path.home() / "open-aea").absolute()
     open_autonomy_dir = Path(
         open_autonomy_dir or Path.home() / "open-autonomy"
     ).absolute()
 
+    if dev_mode:
+        for name, path in (
+            (PACKAGES_DIR, packages_dir),
+            (OPEN_AEA_DIR, open_aea_dir),
+            (OPEN_AUTONOMY_DIR, open_autonomy_dir),
+        ):
+            if not path.exists():
+                flag = "--" + "-".join(name.split("_"))
+                raise click.ClickException(
+                    f"Path does not exist @ {path} for {name}; Please provide proper value for {flag}"
```

Comment:
We're missing a test here.

Reply:
yes that's the only thing worth commenting on, looks good otherwise

Reply:
Addressed https://github.com/valory-xyz/open-autonomy/pull/1403/commits/4a41629a8be1801bbad0cd6a152f654e3895fc73